batch/2.2/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Jakarta Batch 2.2 (Under Development)"
+date: 2025-04-15
+summary: "Enhancements for Jakarta EE 12"
+---
+
+<!-- Please describe the high-level changes made to Jakarta Wombat 1.0. --> 
+<!-- The intent is for the first two sections to be an executive summary in the range of 300 to 800 characters. -->
+<!-- Links can accompany the executive summary, but cannot substitute for an executive summary. -->
+
+This release aims to remove references to SecurityManager.
+
+### New features, enhancements or additions
+<!-- List here -->
+* Remove SecurityManager references from API
+
+### Removals, deprecations or backwards incompatible changes
+<!-- List here -->
+* None
+
+### Minimum Java SE Version
+<!-- Specify the minimum required Java SE version for this specification -->
+**Java SE 21 or higher**
+
+# Details
+
+* [Jakarta Batch 2.2 Release Record](https://projects.eclipse.org/projects/ee4j.batch/releases/2.2.0)
+<!-- 
+The commented lines can be uncommented and updated once the records are available.
+* [Jakarta Wombat 1.0 Specification Document](./jakarta-wombat-spec-1.0.pdf) (PDF)
+* [Jakarta Wombat 1.0 Specification Document](./jakarta-wombat-spec-1.0.html) (HTML)
+* [Jakarta Wombat 1.0 Javadoc](./apidocs)
+* [Jakarta Wombat 1.0 XML Schema](https://jakarta.ee/xml/ns/jakartaee/wombat-1.0.xsd)
+* [Jakarta Wombat 1.0 TCK](http://downloads.eclipse.org/jakarta/wombat/1.0.0/wombat-tck-1.0.0.zip)
+* Maven coordinates
+  * [jakarta.wombat:jakarta.wombat-api:jar:1.0.0](https://search.maven.org/artifact/jakarta.wombat/jakarta.wombat-api/1.0.0/jar) -->
+* Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
```

Comment:
This probably could have stayed commented out.. if this matters.

Reply:
I requested it not be commented out.